### PR TITLE
COR-418 fix(clerk-js): Improve organization invitation form UX

### PR DESCRIPTION
**Pull request by raptisj**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

Improve UX for organization invitation form

https://github.com/clerkinc/javascript/assets/23478420/996f6174-5e40-43a4-8c5e-b822ec34d15f

Screen Shot 2023-06-09 at 11 42 47 AM

### CODE REVIEW

File: packages/clerk-js/src/ui/elements/TagInput.tsx [hunk 1]
```diff
@@ -137,15 +137,18 @@ export const TagInput = (props: TagInputProps) => {
           {tag}
         </TagPill>
       ))}
+
       <Input
         ref={inputRef}
         value={input}
         type='email'
+        data-testid='tag-input'
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/elements/TagInput.tsx:145]: How come we need this?

File: packages/clerk-js/src/ui/elements/TagInput.tsx [hunk 2]
```diff
@@ -93,6 +93,7 @@ export const TagInput = (props: TagInputProps) => {
 
   const handleChange: React.ChangeEventHandler<HTMLInputElement> = e => {
     setInput(e.target.value);
+    // isDirtyField(e.target.value);
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/elements/TagInput.tsx:96]: 🧹

File: .changeset/sharp-houses-study.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+Improve UX for organization invitation form
```
Comment (nikosdouvlis) [inline .changeset/sharp-houses-study.md:5]: Let's add an empty changeset for this one as this message is rather abstract. 
Sorry for being so annoying the past few days with these things, but please bear with me as we're trying to set some guidelines!

**Review by nikosdouvlis — COMMENTED** [review]
@raptisj this looks good but can you please add a screenshot for the reviewers?

File: packages/clerk-js/src/ui/elements/TagInput.tsx (hunk 1 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/elements/TagInput.tsx:144]: What was the reason for this?

Comment (raptisj) [reply packages/clerk-js/src/ui/elements/TagInput.tsx:148]: @desiprisg We removed the placeholder that we were using to access it in the tests.

File: .changeset/tame-mirrors-melt.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+---
+---
```
[the remark this answers is not in the document]
Comment (raptisj) [reply .changeset/tame-mirrors-melt.md:2]: @nikosdouvlis is this correct? Or I shouldn't have any file being created

File: packages/clerk-js/src/ui/elements/TagInput.tsx (hunk 1 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/elements/TagInput.tsx:148]: Can you try using the `type` ?

**Review by desiprisg — APPROVED** [review]

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**